docker/Dockerfile.psga-pipeline

```diff
@@ -6,15 +6,13 @@ RUN apt-get update \
   && rm -rf /var/lib/apt/lists/*
 
 
-# fetch a version of nextflow including the nf-amazon plugin. Install the nf-amazon plugin once for all
-ENV NXF_VER="22.08.0-edge"
+# Install the nf-amazon plugin once for all
+ENV NXF_VER="22.09.7-edge"
```

Comment:
This comes with a later version of the nf-amazon plugin we used. This plugin comes with the sts jar.